Alexander_etal_OpEd_Reanalaysis.tex

```diff
@@ -52,11 +70,11 @@ \section{Background}
 %\footnote{All of the assemblies done by NCGR for MMETSP are absent from the TSA (unless I am not seeing them for some reason, potentially double check \href{https://www.ncbi.nlm.nih.gov/bioproject/248394}{here} and \href{https://www.ncbi.nlm.nih.gov/bioproject/231566}{here}). I would say that it appears that only two transcriptomes were uploaded to the TSA (probably because it is a bit of a PITA to get through their QC system). And thus, iMicrobe. I am not sure how the discussion of this should go but I do think that it proves our point quite nicely.}\footnote{The storage of microbial genomes might be an exception to this rule. NCBI's Microbial Genome Resources and JGI's IMG/M seem to be well adopted by their community.}\footnote{Perhaps a visual comparison of \# of datasets between the SRA and the TSA would be good?}
-In fact, a substantial proportion of these data products might be aptly categorized as `dark data,' as they are largely undiscoverable and often archived independently in association with a publication or on private servers. Even more limiting, however, is that the guidelines for public databases such as the TSA specifically state that ``Assemblies from sequences not directly sequenced by the submitter'' should not be uploaded to the TSA, thereby excluding the potential for reassembled datasets to be made available and directly linked to preexisting BioProjects, BioSamples, TSAs, and SRA entries.
+In fact, a substantial proportion of these data products might be aptly categorized as `dark data,' as they are largely undiscoverable and often archived independently in association with a publication or on private servers. Even more limiting, however, is that the guidelines for public databases such as the TSA specifically state that ``Assemblies from sequences not directly sequenced by the submitter'' should not be uploaded to the TSA, thereby excluding the potential for reassembled datasets to be made available and directly linked to preexisting BioProjects, BioSamples, TSAs, and SRA entries.\footnote{@CTB do we have a citation/link for this policy?}
```

Comment:
Yes, it is in the first paragraph on this page: https://www.ncbi.nlm.nih.gov/genbank/tsa/

I guess we can just link to the website? Seems somewhat static?

Reply:
@ctb Thoughts?